src/OrbitGl/MultivariateTimeSeries.h

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_MULTIVARIATE_TIME_SERIES_H_
+#define ORBIT_GL_MULTIVARIATE_TIME_SERIES_H_
+
+#include <array>
+#include <map>
+#include <optional>
+#include <string>
+
+#include "OrbitBase/Logging.h"
+
+template <size_t Dimension>
+class MultivariateTimeSeries {
+  static_assert(Dimension >= 1, "Dimension must be at least 1");
+
+ public:
+  explicit MultivariateTimeSeries(std::array<std::string, Dimension> series_names)
+      : series_names_(std::move(series_names)) {}
+
+  [[nodiscard]] const std::array<std::string, Dimension>& series_names() const {
+    return series_names_;
+  }
+  [[nodiscard]] const std::map<uint64_t, std::array<double, Dimension>>& time_to_series_values()
+      const {
+    return time_to_series_values_;
+  }
+  [[nodiscard]] double min_series_value() const { return min_; }
+  [[nodiscard]] double max_series_value() const { return max_; }
```

Comment:
nit: `min()` instead of `min_series_value()`. 

From the style guide:
> Regular functions have mixed case; accessors and mutators may be named like variables. Ordinarily, functions should start with a capital letter and have a capital letter for each new word.

These are accessors, so I would name it like the variable.

Reply:
Done! Thanks!